### PR TITLE
[Bug]: Changed user_id to username in recipe

### LINKED ISSUE #81: [Bug]: Change recipe model and router to use username instead of user id

### PR DESCRIPTION
**Pull request by xvxnoah**

### CODE REVIEW

**Review by andreu-vall — DISMISSED** [review]
Working perfectly and makes more sense to use the username as it is already unique 👌 

We will probably have to delete the recipes created before this point right?

**Review by andreu-vall — APPROVED** [review]
Ups I dismissed my review by adding a commit LOL. Anyway I should approve again...

Working perfectly and makes more sense to use the username as it is already unique 👌

We will probably have to delete the recipes created before this point right?

**Comment by xvxnoah** [comment]
> Working perfectly and makes more sense to use the username as it is already unique 👌 
> 
> 
> 
> We will probably have to delete the recipes created before this point right?

Yep, same thing when we approve the merge of the reviews, since I changed the model of recipe and now it will include the reviews.

**Pull request merged**
